lib/service/lokal-hamburk.js

```diff
@@ -23,19 +23,22 @@ module.exports = function () {
   var processMenu = function (obj, options, next) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l23)